components/phpunit_bridge.rst

```diff
@@ -30,6 +30,8 @@ Installation
 
     $ composer require --dev "symfony/phpunit-bridge:*"
```

Comment:
Do we want the „:*“ ?

Reply:
I would remove the start, relying on `composer require` to pick a good version constraint for us

Reply:
@OskarStark @wouterj so I don't need to remove the star for now?

Reply:
Please remove the star. Using a start would result in `"symfony/phpunit-bridge": "*"` being added to your `composer.json`, which is a bit scary. Without the star, the command figures out the version constraint so it'll end up as `^4.2` or so in the `composer.json`, which is better imo